### ISSUE TITLE
Missing information on new permissions

### ISSUE THREAD

**Issue by ramschmaerchen** [WhisperSystems/TextSecure#3860]
The new version needs a lot of new permissions without proper explanation.

**Comment by 2-4601** [comment]
[Duplicate](https://github.com/WhisperSystems/TextSecure/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aclosed+permissions) of #3782, #3781, #3771.

**Comment by ramschmaerchen** [comment]
Bug has not been fixed, so I guess this issue is still valid.

**Comment by 2-4601** [comment]
What bug?

**Comment by ramschmaerchen** [comment]
No information given

**Comment by 2-4601** [comment]
No information taken

**Comment by ramschmaerchen** [comment]
@2-4601 I really don't know what to do with your answers. You seem to not take any of the people talking about this issue seriously. Not openly providing information about changes in data usage is very concerning. I can only hope Open Whisper Systems will fix this obvious flaw in User Interaction Design sometime soon for the sake of this software's reputation.

**Comment by 2-4601** [comment]
I really don't know what to do with your questions. You don't seem to take this issue tracker seriously.
- You created a new issue without searching for existing issues
- There's a big yellow banner when you create a new issue explaining [how this place works](https://github.com/WhisperSystems/TextSecure/blob/master/contributing.md)
- You were given information about the existing issues which provide an answer to your question and what is the proper place to ask questions such as yours

> Not openly providing information about changes in data usage is very concerning.

This is an open source project. The information you are seeking is **in this very place you are asking for it.** Why are you not taking it?

**Comment by ramschmaerchen** [comment]
Yes, this would be a good place to provide detailed information about data usage and permissions, but I don't see the information given. I don't see it in the Readme, I don't see any FAQ, I don't even see this information on this application's website. I only was able to find some information in the forums, but this information did not seem to be complete. To take things further, the best place would be to post  information about data usage and needed permissions at the place where people install the software: Google Play Store. The least thing Open Whisper Systems could do is provide a link to a detailed explanation about why all those permissions are needed there.

Yes, one could look up the source but keep in mind that not even skilled people can look into source code because of multiple reasons:
- Unfamiliar with the language
- Unfamiliar with the OS
- Unfamiliar with some of the libraries
- Very time consuming
- Sometimes unclear (depends on coding style and rules)

There is no way unskilled people could look into the source.

I know it's Open Source and one should do PR instead of issue requests, but to do a meaningful doc-PR I would need to spend days browsing the source code and getting familiar with why and how it works. I am sure there are people around here who can write detailed information about data usage and permissions in one hour because they are way familiar with this project more than I am.

If you could do it, that would be much appreciated.


So, let me rephrase my issue report:

When I want to upgrade to my Text Secure Version 2.22.2 to Text Secure Version 2.23.3 in the google play store, a lot of new permissions need to be given. They are as follows:
* Device and App history
* Calendar
* Location
* Camera
* Microphone
* Wi-Fi connection information

I was able to aqucire information on why some of those permissions are needed, but it took a long time to gather this information. Additionally, I am unsure if all of this information is correct, as some answers were given by people not transparently connected to Open Whisper Systems.

To benefit open communication and good User Interaction Design, I kindly ask for detailed information about new permissions at the place, where the application gets updated: The Google Play Store. As some information may take a lot of space in the application's update details, I guess a link to this information should be sufficient.

Currently installed app version: TextSecure 2.22.2
